modules/node_modules/@ncigdc/utils/formatFileSize.js

```diff
@@ -1,31 +1,4 @@
 // @flow
+import filesize from 'filesize';
```

Comment:
We show `51 KB`


osx shows

![image](https://cloud.githubusercontent.com/assets/743976/24432156/2f5cbace-13ee-11e7-8b38-40ff46ad21a1.png)